src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -157,6 +158,67 @@ func (a *AdminAPI) sendOne(method, path string, body, into interface{}) error {
 	return maybeUnmarshalRespInto(method, url, res, into)
 }
 
+// sendAll sends a request to all URLs in the admin client. The first successful
+// response will be unmarshaled into into if it is non-nil.
+//
+// As of v21.11.1, the Redpanda admin API forwards requests to the leader based
```

Comment:
Probably being picky on terminology, but for the benefit of anyone reading the comment in future: we aren't forwarding these, we're redirecting.

Certain actions actually do forward internally, which aren't problematic -- it's just redirects that are problematic.